viewer/src/main/java/nl/b3p/viewer/stripes/AttributesActionBean.java

```diff
@@ -97,6 +97,9 @@
     @Validate
     private List<String> attributesToInclude = new ArrayList();
 
+    /**
+     * To force returning geometry set this to true in the request.
```

Comment:
I think this only works when you set the geometry in attributesToInclude.
I think it's better to change the variablename to reflect it's purpose.